nexus/types/src/deployment.rs

```diff
@@ -174,6 +174,21 @@ pub struct Blueprint {
     // See blueprint execution for more on this.
     pub external_dns_version: Generation,
 
+    /// The minimum release generation to accept for target release
+    /// configuration. Target release configuration with a generation less than
+    /// this number will be ignored.
+    ///
+    /// For example, let's say that the current target release generation is 5.
+    /// Then, when reconfigurator detects a MUPdate:
+    ///
+    /// * the target release is ignored in favor of the install dataset
+    /// * this field is set to 6
+    ///
+    /// Once the target release generation is updated to 6 or higher,
```

Comment:
The wording on this is correct but I read it incorrectly the first time through, because "the target release generation" is pretty similar to the name of this field; I was confused about how the target release generation might be set to 6 in the future when we've already set this field to 6. The event we're waiting for here is "an operator sets a new target release" (which will necessarily bump the generation number up to 6), right? And we also have to have cleared all the mupdate overrides in the meantime (or as a part of that new release being set)?

Reply:
I've updated it to have a more active voice:

```
    /// Once an operator sets a new target release, its generation will be 6 or
    /// higher. Reconfigurator will then know that it is back in charge of
    /// driving the system to the target release.
```

> And we also have to have cleared all the mupdate overrides in the meantime (or as a part of that new release being set)?

I believe that's right, yes.